packages/api-key-service/src/services/api-key.ts

```diff
@@ -14,6 +16,39 @@ export const updateApiKey = (apiKey: ApiKeyModel, dto: Partial<ApiKeyModel>): Pr
   )
 }
 
-export const getApiKey = (apiKey: string): Promise<ApiKeyModel | undefined> => {
-  return db.get(Object.assign(new ApiKeyModel(), { apiKey }))
+export const getApiKey = (apiKey: Partial<ApiKeyModel>): Promise<ApiKeyModel | undefined> => {
+  return db.get(Object.assign(new ApiKeyModel(), apiKey))
+}
+
+export const batchGetApiKeys = async (
+  customer: LoginIdentity & { developerId: string },
+  startKey?: { [s: string]: string },
```

Comment:
Is it worth defining this as a global type ie 

```ts
interface StringMap {
  [s: string]: string
}
```

Reply:
Yea why not

Reply:
changed to partial of apiKeyModel for consistency of type